torch/_logging/_registrations.py

```diff
@@ -11,6 +11,8 @@
 register_artifact("bytecode", off_by_default=True)
 register_artifact("graph")
 register_artifact("graph_code")
+register_artifact("graph_source", off_by_default=True)
+register_artifact("starts_line")
```

Comment:
I'm not sure this is a very explicable name for end users. How about something like "source_trace"?